app/assets/stylesheets/custom.scss

```diff
@@ -291,3 +288,31 @@ ul, menu, dir {
 .filter {
   text-align: center;
 }
+
+// /* =========================
+//
+//      USER LISTINGS INDEX PAGE
+//
+//    =========================*/
+
+#img_container {
+  position:relative;
+  display:inline-block;
+  text-align:center;
+}
+
+.edit-button {
+  position:absolute;
```

Comment:
Properties should be ordered bottom, height, position, right, width<br>Colon after property should be followed by one space